### PR TITLE
Use Integer instead of Fixnum

### PR DESCRIPTION
**Pull request by brookskindle**
Fixnum was deprecated in ruby v2.4.0, they're called Integers now.

### CODE REVIEW

File: lessons/day1/ruby_types.md [hunk 1]
```diff
@@ -21,11 +21,11 @@ To determine the type of an object you can use `.class`
...
+puts x.class #Integer
 
 word = "hello"
 puts word.class #String
 ```
 
 ### Resources
-* [Ada Ruby Types Video](https://adaacademy.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=0a3aad23-0e7d-4dca-b6bb-f1ed3bed8692) (5:52)
\ No newline at end of file
+* [Ada Ruby Types Video](https://adaacademy.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=0a3aad23-0e7d-4dca-b6bb-f1ed3bed8692) (5:52)
```
Comment (brookskindle) [inline lessons/day1/ruby_types.md:36]: Not sure why this line shows as being edited - I don't see any difference.

I did make this change via the github UI, maybe that added/removed a trailing newline?

**Comment by susanev** [comment]
Hi Brooks, can you remove the reference to "whole numbers"? Whole numbers (0, 1, 2, 3 ...) are not the same as integers, maybe just provide a list (..., -3, -2, -1, 0, 1, 2, 3 ... ) if you want to give an example.

**Comment by susanev** [comment]
Oh also, can you add a reference somewhere that mentions what Fixnum was, and what version it was removed from; I am sure they will still see that reference in online resources so I think its useful to let them know about it.

**Comment by brookskindle** [comment]
Sure I can add those suggestions

**Pull request merged**